src/Tests/Cluster/RemoteInfo/RemoteInfoApiTests.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.Collections.Generic;
+using Elasticsearch.Net;
+using FluentAssertions;
+using Nest;
+using Tests.Framework;
+using Tests.Framework.Integration;
+using Tests.Framework.ManagedElasticsearch.Clusters;
+using Tests.Framework.MockData;
+using Xunit;
+using M = System.Collections.Generic.Dictionary<string, object>;
+using static Nest.Infer;
+
+namespace Tests.Cluster.RemoteInfo
+{
+	public class RemoteInfoApiTests : ApiIntegrationTestBase<ReadOnlyCluster, IRemoteInfoResponse, IRemoteInfoRequest, RemoteInfoDescriptor, RemoteInfoRequest>
+	{
+		protected override void IntegrationSetup(IElasticClient client, CallUniqueValues values)
+		{
+			var enableRemoteClusters = client.ClusterPutSettings(new ClusterPutSettingsRequest
+			{
+				Transient = new M
```

Comment:
I think it's be better to use the type here instead of the `M` alias, for documentation purposes